modules/transport.js

```diff
@@ -617,6 +625,11 @@ Transport.prototype.shared = {
 	 * @todo Add description of the function
 	 */
 	postSignature(query, cb) {
+		if (!library.config.broadcasts.active) {
+			return library.logger.debug(
+				'Receiving signatures disabled by user through config.json'
+			);
```

Comment:
If `library.config.broadcasts.active` is false, the callback will never be called because it will return from the function... This could cause issues if the caller relies on the callback; not sure if we should call with an error in this case (since broadcast is blocked) but it should be called in any case.

It's only a problem for the singular form of the function because the plural ones don't have a callback.

Reply:
Actually isn't this meant to be added to the postSignatures (plural) since it's for the WebSocket layer?